xds/src/main/java/io/grpc/xds/EnvoyProtoData.java

```diff
@@ -195,6 +196,14 @@ public String toString() {
     private final int loadBalancingWeight;
     private final boolean isHealthy;
 
+    @VisibleForTesting
+    LbEndpoint(String address, int port, int loadBalancingWeight, boolean isHealthy) {
+      this(
+          new EquivalentAddressGroup(
+              ImmutableList.<SocketAddress>of(new InetSocketAddress(address, port))),
```

Comment:
List is not needed, you can just use the constructor `EquivalentAddressGroup(SocketAddress addr)`.

Reply:
Done.